### ISSUE TITLE
Installation error with react-intl-toolkit 

### ISSUE THREAD

**Issue by nitish-mehta** [zation/react-intl-toolkit#1]
While installing the package, it failed on my linux system with below error. 

```
npm ERR! Linux 4.4.0-66-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "react-intl-toolkit" "--save-dev"
npm ERR! node v6.10.0
npm ERR! npm  v3.10.10
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! react-intl-toolkit@0.0.2 install: `babel src/ --out-dir ./`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the react-intl-toolkit@0.0.2 install script 'babel src/ --out-dir ./'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-intl-toolkit package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     babel src/ --out-dir ./
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-intl-toolkit
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-intl-toolkit
npm ERR! There is likely additional logging output above.

```

My node version is 6.x & npm is 3.x

**Comment by nitish-mehta** [comment]
It seems that there is a dependency to `babel-cli `, which if is missing , results in this issue.
I would suggest updating the docs to make it clearer.

Works now for me post installing babel-cli

**Comment by zation** [comment]
Thanks for the feedback! I moved `babel-cli` from `devDependencies` to `dependencies`. The new version is published as 0.0.3
